### PR TITLE
Lavaland stuff.

### PR DESCRIPTION
**Pull request by TheSadFox**
Changed swoop attack icon to a one that actually exists. Ash drakes swoop and bubblegums charge landing locations should be visable now. Whoever the monkey is that set it to use a non existant icon, you owe me so many rounds.

Removed two outcomes of drinking dragons blood - skeleton transformation and nothing. Getting big nothing after killing the drake is fairly shit. Doubt dragon transformation will become too common, especially once Mac's PR making it so only one drake can spawn will go through.

Replaced soul stone in necropolis chests with a lesser necromantic stone. Its basically a necro stone that can only sustain one thrall at a time and doesn't spawn them with claymores. Will think later about some other equipment the thrall should get. Soul stones were shit anyways.

Added burn and toxin medkits to the vendor.
Added standard plasma cutter to the vendor for 2500 points.
Lowered plasma blasts damage from 5 to 4 (from 20 to 16 in low pressure environment).
Increased plasma cutters and advanced plasma cutters melee damage from 12 to 15 and from 15 to 18.
Lowered cost of silver and diamond pickaxes from whatever their retarded prices were to something that might actually make people buy them.
Moved toys, cigars and stuff to the bottom of vendors list.

Removed runed metal from chest loot.

### CODE REVIEW

**Comment by TheSadFox** [comment]
Such long commit much wow i'm retarded.

**Comment by Super3222** [comment]
Long commit messages are good when you supply information. It helps with debugging / figuring out when Y was implemented into X.

File: code/game/gamemodes/wizard/artefact.dm [hunk 1]
```diff
@@ -195,7 +195,50 @@
 	H.equip_to_slot_or_del(new /obj/item/weapon/claymore(H), slot_r_hand)
 	H.equip_to_slot_or_del(new /obj/item/weapon/twohanded/spear(H), slot_back)
 
+//Lesser Necromantic Stone - basically a nerfed necro stone that can handle only one soul and cannot be used to farm claymores, intended for necropolis chests.
+/obj/item/device/necromantic_stone/lesser
+	name = "lesser necromantic stone"
 
+/obj/item/device/necromantic_stone/attack(mob/living/carbon/human/M, mob/living/carbon/human/user)
```
Comment (RemieRichards) [inline code/game/gamemodes/wizard/artefact.dm]: Delete this, give `/obj/item/device/necromantic_stone` a variable called `max_spookies` and set it to 3 by default, 1 for your new subtype, and replace the `if(spooky_scaries.len >= 3 && !unlimited)` with `if(spooky_scaries.len >= max_spookies && !unlimited)`

**Comment by RemieRichards** [comment]
Yep that's exactly what I asked (and a bit more by remembering to update the messages too!) Good work :+1:

File: code/game/gamemodes/wizard/artefact.dm [hunk 2]
```diff
@@ -195,8 +196,23 @@
 	H.equip_to_slot_or_del(new /obj/item/weapon/claymore(H), slot_r_hand)
 	H.equip_to_slot_or_del(new /obj/item/weapon/twohanded/spear(H), slot_back)
 
+//Lesser Necromantic Stone - basically a nerfed necro stone that can handle only one soul and cannot be used to farm claymores, intended for necropolis chests.
+/obj/item/device/necromantic_stone/lesser
+	name = "lesser necromantic stone"
+	max_spookies = 1
 
 
+/obj/item/device/necromantic_stone/proc/equip_roman_skeleton_lesser(mob/living/carbon/human/H)
```
Comment (RemieRichards) [inline code/game/gamemodes/wizard/artefact.dm]: You'll actually want to swap `/proc/` here for `/lesser/` and change this procs name to the original proc's `equip_roman_skeleton`, that way everything works properly and cleanly.

**Comment by TheSadFox** [comment]
I usually have a lot of ideas going through my head but this week i'm a creative desert. I will just change one thing in a moment and this will be ready for merge. Once i come up with ideas for more loot i'll make a new PR.

**Comment by TheSadFox** [comment]
T'is ready to be merged now.

We don't really have any medicore swords/blades that i could substitute claymore with, so instead the thralls starts with a bone knife and spear. Not like the items matter anyways becouse the stone will most of the time be used by miners on miners, who will ditch all that crap instantly anyways to use their own items.

You can now also thrall people who are knocked out, instead of just dead ones. Its something that annoyed me greatly during my wizard rounds where i used it, kill the guy and he instantly ghosts out of the corpse, leaving you unable to thrall them/forces you to yell/whisper at him to get back into his fucking body.

Considering that nobody voiced any concerns in the three days since this PR was opened, please don't let this rot and merge it soon, lack of the marker for bubblegum is annoying me greatly (this is also a lesson that i should create separate PR's for shit like this yeah).

**Comment by TheSadFox** [comment]
Updated OP with new changes.

If anyone wondered why did i add toxin kit to the vendor - new watchers are fucking disgusting. While they are not really dangerous, after fightning them you are left with a ton of bolamine in your bloodstream. Miners can pay 600 points now to help get rid of it and not rely on legion hearts for that.

Plasma cutter - Its for those rounds when you can't count for RnD to make you hyper KA's. You pay 2500 points for an ability to get through rocks and get to tendrils/create an arena to fight megafauna a bit faster. This changes nothing during rounds in which RnD is actually doing their job as you will get much more efficient shit from them. Lowered its ranged damage, before it was pretty much as effective as standard KA. If you think old value was fine, let me know and i'll revert this.
People might probably start dual wielding KA in one hand and a cutter in another to speed up killing normal fauna, but then again, normal fauna is hardly an issue anyways.
Increased melee damage as i did feel that a fucking knife/welding tool being stronger/same as these was kinda silly. Don't think it will cause any balance issues, miners already get a 15 force weapon and RnD can make better gear anyways. 

Pickaxes were pretty much never bought becouse of how overpriced they were.

**Comment by TheSadFox** [comment]
Uh, forgot to make a changelog for this, i'll make a PR for it later, i guess.

**Pull request merged**